src/booleans.js

```diff
@@ -1,61 +1,66 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a&&b
 };
 
 function either(a, b) {
-  // your code here
+  return a||b
 };
 
 function none(a, b) {
-  // your code here
+  return !a&&!b
 };
 
 function one(a, b) {
-  // your code here
+  return (a&&!b || !a&b)? true:false;
 };
 
 function truthiness(a) {
-  // your code here
+  return a? true: false;
```

Comment:
The logic here is fine, but it can be condensed by using two NOT operators (!!) together to create a double negation. What this does is to first evaluate the value to a boolean that will be the opposite of what it actually is, eg passing in a string (which is a truthy value) will be made `false` by one of the ! operators, then the second ! will flip that to `true`, and so evaluating the string as being truthy. Nice trick that gets used more often than you might think!
```
  return !!a
```